src/Build/BackEnd/Components/RequestBuilder/IntrinsicTasks/ItemGroupLoggingHelper.cs

```diff
@@ -203,5 +216,56 @@ private static void AppendStringFromParameterValue(ReuseableStringBuilder sb, ob
                 ErrorUtilities.ThrowInternalErrorUnreachable();
             }
         }
+
+        internal static void LogTaskParameter(
+            LoggingContext loggingContext,
+            TaskParameterMessageKind messageKind,
+            string itemType,
+            IList items,
+            bool logItemMetadata)
+        {
+            var args = CreateTaskParameterEventArgs(loggingContext.BuildEventContext, messageKind, itemType, items, logItemMetadata, DateTime.UtcNow);
+            loggingContext.LogBuildEvent(args);
+        }
+
+        internal static TaskParameterEventArgs CreateTaskParameterEventArgs(
+            BuildEventContext buildEventContext,
+            TaskParameterMessageKind messageKind,
+            string itemName,
```

Comment:
nit: inconsistency between itemName and itemType. Personally I prefer itemType since it seems to better distinguish from itemSpec

Reply:
itemType feels like it's something about the type of the item, like "value type" or "string" or "ITaskItem". itemName is less ambiguous so I'll unify on itemName.

Reply:
actually @cdmihai you are right, from reading the code it looks like the canonical term for this is `ItemType` as seen here https://source.dot.net/#Microsoft.Build/Instance/ProjectItemInstance.cs,af383bff89575c65.

Should I go and rename itemName to itemType everywhere?

Reply:
It would probably settle it once and for all. Unless there exists another public API that uses 'ItemName` :)